esp/xslt/esdl2monitor.xslt

```diff
@@ -45,8 +44,9 @@
 EXPORT <xsl:value-of select="$definitionName"/>() := MACRO<xsl:text>
 </xsl:text>
 </xsl:if>
+
 <xsl:if test="$diffmode='Monitor'">
-IMPORT cassandra;
+IMPORT mysql;
```

Comment:
Note that Cassandra support has been moved to a different file: `esdl2monitor_cassandra.xslt`